lib/ansible/modules/cloud/xenserver/xenserver_guest.py

```diff
@@ -28,7 +28,7 @@
 - 'To acquire XenAPI Python library, just run C(pip install XenAPI) on your Ansible Control Node. The library can also be found inside
    Citrix Hypervisor/XenServer SDK (downloadable from Citrix website). Copy the XenAPI.py file from the SDK to your Python site-packages on your
    Ansible Control Node to use it. Latest version of the library can also be acquired from GitHub:
-   https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI.py'
+   https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py'
```

Comment:
```suggestion
   U(https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py)'
```
(Same for the ones below.)

Reply:
In the docs I see that URLs are already automatically detected and converted to hyperlinks. I'm not sure if `U()` makes any difference.

Reply:
IMO it's bad style to rely on implicit undocumented conversions where there is an explicit way of doing it.